Composite-KEM-2023.asn

```diff
@@ -282,4 +283,23 @@ kema-MLKEM1024-X448-KMAC256 KEM-ALGORITHM ::=
       id-MLKEM1024-X448-KMAC256, 
       pk-MLKEM1024-X448-KMAC256 }
 
+
+--
+-- Expand the S/MIME capabilities set used by CMS [RFC5911]
+--
+
+SMimeCaps SMIME-CAPS ::=
+    { kema-MLKEM512-ECDH-P256-KMAC128 |
```

Comment:
I don't know anything about ASN.1 but shouldn't these be `kema-MLKEM512-ECDH-P256-KMAC128.&smimeCaps`, etc.  I just base this on what I see in RFC 5911.

Reply:
Yup, sure should. Good catch.